contexts/credentials/examples/v1

```diff
@@ -20,11 +20,15 @@
     "UniversityDegreeCredential": "ex:UniversityDegreeCredential",
     "ZkpExampleSchema2018": "ex:ZkpExampleSchema2018",
 
-    "alumniOf": {"schema:alumniOf", "@type": rdf:HTML},
+    "alumniOf": {"@id": "schema:alumniOf", "@type": "rdf:HTML"},
```

Comment:
Based on a discussion with @chaals today, this is going to be changing to a scoped context that supports language expressions. I don't have the details yet, have to talk w/ @dlongley about it, but it will most likely be a scoped context that aliases @value, @language, and @dir to `value`, `language`, and `dir` respectively (and presumes that @dir is going to be a special JSON-LD 1.1 keyword.

Reply:
@msporny Got it, sounds good.
Although - can we merge this change now (so that at least the context doesn't error out), and do the language/dir change separately?